### DISCUSSION TITLE
Slider marks and height

### DISCUSSION THREAD

**Discussion by srweeks** [mantinedev/mantine#4078] [Q&A] [answered]
Hello, 

Wondering if I'm missing something obvious the container is set to a height of 8px so any following components overlay the marks e.g., https://codesandbox.io/s/wonderful-water-cfnrvp?file=/src/App.tsx.

Is there an obvious way we should manage this issue without having to enforce a height?

Thanks!

**Comment by rtivital** [comment, marked as answer]
Slider marks have absolute position. It is hard to predict which values you will put there, and that is why you need to offset them on your side. This can be done by setting margin-bottom styles – https://codesandbox.io/s/serverless-dream-svx4lm

**Comment by srweeks** [comment]
thanks
